### PR TITLE
fix: support MINIO_ROOT* envs

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: pkg/credentials/env_minio.go [hunk 1]
```diff
@@ -22,10 +22,12 @@ import "os"
 // A EnvMinio retrieves credentials from the environment variables of the
 // running process. EnvMinioironment credentials never expire.
 //
-// EnvMinioironment variables used:
+// Environment variables used:
 //
 // * Access Key ID:     MINIO_ACCESS_KEY.
```
Comment (vadmeste) [inline pkg/credentials/env_minio.go:27]: Shouldn't we remove the deprecated env vars ?

**Review by vadmeste — COMMENTED** [review]
One comment

File: pkg/credentials/env_minio.go (hunk 1 above)
Comment (harshavardhana) [reply pkg/credentials/env_minio.go:27]: No not everyone is expected to migrate, and we shall never remove these ENVs from MinIO either @vadmeste

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by kannappanr — APPROVED** [review]

**Pull request merged**